examples/hf_trainer_api/hf_image_classification/image_classification.py

```diff
@@ -416,7 +423,10 @@ def val_transforms(example_batch):
             checkpoint = training_args.resume_from_checkpoint
         elif last_checkpoint is not None:
             checkpoint = last_checkpoint
-        train_result = trainer.train(resume_from_checkpoint=checkpoint)
+
+        # TODO: use the actual steps_completed and not this placeholder.
+        with dsat.dsat_reporting_context(core_context, op=det_callback.current_op):
+            train_result = trainer.train(resume_from_checkpoint=checkpoint)
```

Comment:
TODO or not TODO? what's the status on this one? seems not to be using steps_completed

Reply:
@garrett361 I think we fixed this right?

Reply:
Yeah I think this is old, removing.

Reply:
Yep, old , also missed because I forgot to look outside of `/dsat` for todos.